R/ari_stitch.R

```diff
@@ -74,44 +74,40 @@ ari_stitch <- function(images, audio,
                        output = tempfile(fileext = ".mp4"),
                        verbose = FALSE,
                        cleanup = TRUE,
-                       ffmpeg_opts = "",
-                       divisible_height = TRUE,
-                       audio_codec = get_audio_codec(),
-                       video_codec = get_video_codec(),
-                       video_sync_method = "2",
-                       audio_bitrate = NULL,
-                       video_bitrate = NULL,
-                       pixel_format = "yuv420p",
-                       fast_start = FALSE,
-                       deinterlace = FALSE,
-                       stereo_audio = TRUE,
+                       ffmpeg_args = list(frames_per_second = NULL,
+                                          video_filters = NULL,
+                                          divisible_height = TRUE,
+                                          audio_codec = get_audio_codec(),
+                                          video_codec = get_video_codec(),
+                                          deinterlace = FALSE,
+                                          ffmpeg_opts = "",
+                                          audio_bitrate = NULL,
+                                          video_bitrate = NULL,
+                                          video_sync_method = "2",
+                                          pixel_format = "yuv420p",
+                                          fast_start = FALSE,
+                                          stereo_audio = TRUE),
```

Comment:
Need to create a new function called ffmpeg_defaults():

```r
ffmpeg_defaults <- function(){
list(frames_per_second = NULL,
                                          video_filters = NULL,
                                          divisible_height = TRUE,
                                          audio_codec = get_audio_codec(),
                                          video_codec = get_video_codec(),
                                          deinterlace = FALSE,
                                          ffmpeg_opts = "",
                                          audio_bitrate = NULL,
                                          video_bitrate = NULL,
                                          video_sync_method = "2",
                                          pixel_format = "yuv420p",
                                          fast_start = FALSE,
                                          stereo_audio = TRUE)
}
```

Question for you is do you want this function to take arguments? (Could be something you don't get around to until the future.)